src/components/layers/basemaps/BasemapList.js

```diff
@@ -11,13 +11,10 @@ const styles = {
     },
 };
 
-const BasemapList = (
-    { classes, selectedID, basemaps, selectBasemap },
-    { map }
-) => (
+const BasemapList = ({ classes, selectedID, basemaps, selectBasemap }) => (
     <div className={classes.container} data-test="basemaplist">
         {basemaps
-            .filter(basemap => map.hasLayerSupport(basemap.config.type))
+            // .filter(basemap => map.hasLayerSupport(basemap.config.type))
```

Comment:
Does this change functionality...?

Reply:
No, this change was for Maps GL, and won't change the current functionality.